server/lib/version.js

```diff
@@ -19,89 +19,122 @@
  */
 
 var cp = require('child_process');
+var Bluebird = require('bluebird');
```

Comment:
this should probably be `var Promise = require('bluebird');` to be consistent with other code like this in the project.